hazelcast/src/main/java/com/hazelcast/spi/impl/AbstractInvocationFuture.java

```diff
@@ -147,7 +147,7 @@ public final V join() {
     }
 
     @Override
-    public final V get() throws InterruptedException, ExecutionException {
+    public V get() throws InterruptedException, ExecutionException {
```

Comment:
do you really want to remove final? Seems like for debugging purposes.

Reply:
Same as above. I will replace this part, just looking for more efficient solution